pdns/dnsdistdist/docs/rules-actions.rst

```diff
@@ -622,6 +625,7 @@ These ``DNSRule``\ s be one of the following items:
 
   :param NetMaskGroup nmg: The NetMaskGroup to match on
   :param bool src: Whether to match source or destination address of the packet. Defaults to true (matches source)
+  :param bool quiet: Do not return the list of matched netmasks. Default is false.
```

Comment:
Perhaps we could replace 'return' with 'display', although I see we already use 'return' for the SMN version:
```suggestion
  :param bool quiet: Do not display the list of matched netmasks. Default is false.
```

Reply:
I'll change both!